wafer/talks/models.py

```diff
@@ -283,7 +283,7 @@ def get_authors_display_name(self):
             key=lambda author: u'' if author == self.corresponding_author
                                else author.userprofile.display_name())
         names = [author.userprofile.display_name() for author in authors]
-        if len(names) <= 2:
+        if len(names) <= settings.WAFER_SCHEDULE_MAX_AUTHORS:
             return u' & '.join(names)
```

Comment:
As a future improvement, it would be nice to tweak the formatting here for more than two names so it returns
Author A, Author B, Author C & Author D
instead of
Author A & Author B & Author C & Author D